spec/requests/api/v1/moves_controller_show_spec.rb

```diff
@@ -5,7 +5,7 @@
 RSpec.describe Api::V1::MovesController, :with_client_authentication, :rswag do
   let(:headers) { { 'CONTENT_TYPE': content_type }.merge(auth_headers) }
   let(:content_type) { ApiController::CONTENT_TYPE }
-  let(:move) { create :move }
+  let(:move) { create :move, reason: build(:reason) }
```

Comment:
👍 for using build instead of create!

just double check if we really need it, we do not need specify other associations (but this)?